src/event-manager.js

```diff
@@ -42,7 +43,7 @@ const DEFAULT_OPTIONS = {
   // custom recognizers
   recognizers: null,
   // Manager class
-  Manager,
+  Manager: Hammer ? Hammer.Manager : HammerManagerMock,
```

Comment:
Looking at the code in this PR, seems like `Hammer` is always `null`??
Are you somehow swapping `hammer.browser.js` in for `utils/hammer.js` via Webpack or something? You should make this strategy clear in a comment along with the logic that gets one or the other module at runtime (i.e. here).

Reply:
Yes, using the `browser` field in package.json. I will add a comment.

Reply:
Does this strictly limit usage to only Webpack? I am using the Parcel bundler.